### PR TITLE
fix(shims): moving to the new style modle api format

### LINKED ISSUE #79: Migrate away from old style shims
I recently upgraded to the ember shims beta that uses the new module import syntax, and was getting deprication errors due to the use of things like [`import Component from 'ember-component';`](https://github.com/knownasilya/ember-toggle/blob/master/addon/components/x-toggle-label/component.js#L1)

I suggest that we switch to using the default global Ember import, and then we could possibly do the constant destructuring. What are you're thoughts @knownasilya @rwwagner90 ?

### PR DESCRIPTION
**Pull request by webark**
Using the new ember-cli-babel 6.6 rather than a beta version of the shims.

Closes #79

### CODE REVIEW

**Comment by webark** [comment]
@knownasilya @rwwagner90 when I went to look at the dummy app, it looks like only the default theme is showing up. Any ideas why?

File: tests/dummy/app/index.html [hunk 1]
```diff
@@ -3,14 +3,14 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>ember-toggle</title>
+    <title>Dummy</title>
```
Comment (knownasilya) [inline tests/dummy/app/index.html]: Probably want to revert this one

File: tests/dummy/config/environment.js [hunk 2]
```diff
@@ -51,8 +44,7 @@ module.exports = function(environment) {
   }
 
   if (environment === 'production') {
-    ENV.locationType = 'hash';
-    ENV.rootURL = '/ember-toggle/';
```
Comment (knownasilya) [inline tests/dummy/config/environment.js]: Also all the changes in this file

File: testem.js [hunk 3]
```diff
@@ -3,10 +3,17 @@ module.exports = {
...
-  ]
+  ],
+  browser_args: {
+    Chrome: [
+      '--disable-gpu',
+      '--headless',
+      '--remote-debugging-port=9222',
+      '--window-size=1440,900'
+    ]
+  }
```
Comment (knownasilya) [inline testem.js]: nice, time to write tests.

File: tests/dummy/config/environment.js (hunk 2 above)
Comment (webark) [inline tests/dummy/config/environment.js]: ya, I saw those. Oh. that is for the git hub pages.. Ah. Ok cool.

File: tests/dummy/config/environment.js [hunk 4]
```diff
@@ -21,13 +21,6 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
-    },
-
-    'ember-toggle': {
-      //includedThemes: ['light', 'ios', 'flip'],
-      //excludedThemes: ['flip'],
-      //defaultTheme: 'light',
-      //defaultSize: 'small'
```
Comment (webark) [inline tests/dummy/config/environment.js]: @knownasilya is there a reason why we want to keep this commented out block?

Comment (knownasilya) [inline tests/dummy/config/environment.js]: We can remove that I think.

**Comment by webark** [comment]
@knownasilya I think all the changes you requested have been made. My only concern is that it seems like only the default theme is included in the dummy app. When I spin it up, that is all I am seeing.

**Comment by knownasilya** [comment]
Published alpha.7

**Comment by knownasilya** [comment]
v5 released as well.

**Pull request merged**